lib/lbt/analyzer/XMLTemplateAnalyzer.js

```diff
@@ -304,14 +304,49 @@ class XMLTemplateAnalyzer {
 
 	_analyzeChildren(node) {
 		if ( Array.isArray(node.$$) ) {
-			node.$$.forEach( (child) => this._analyzeNode( child ) );
+			node.$$.forEach( (child) => {
+				// add parentNode reference to child to be able to walk up the parent chain
+				child.$parentNode = node;
+				return this._analyzeNode( child);
+			});
+		}
+	}
+
+	/**
+	 * Walks up the parent chain until fnMatch returns either <code>true</code> or there is no more parent
+	 *
+	 * @param {object} node xml node
+	 * @param {Function} fnMatch matcher function, gets node as argument and returns whether or not the current node matches
+	 * @returns {boolean} <code>true</code> if fnMatch finds a match, <code>false</code> otherwise
+	 * @private
+	 */
+	_findInParentChain(node, fnMatch) {
+		if (fnMatch(node)) {
+			return true;
+		}
+		if (node.$parentNode) {
+			return this._findInParentChain(node.$parentNode, fnMatch);
 		}
+		return false;
 	}
 
 	_analyzeCoreRequire(node) {
 		const coreRequire = getAttributeNS(node, XMLVIEW_CORE_REQUIRE_ATTRIBUTE_NS);
 		let requireContext;
 		if ( coreRequire ) {
+			// expression evaluation
+			if (coreRequire.startsWith("{=")) {
```

Comment:
I think we should not ignore those nodes in general. Otherwise we skip nodes that might be parsable or we still log an error in case there's a new syntax that doesn't start with `{=`.

Reply:
yes I see

I just thought that other parsing errors will then get swallowed just like:
missing comma:

```xml
core:require="{
	Foo:'sap/ui/Foo'
	Bar:'myApp/Bar'
}"
```